src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -45,16 +46,17 @@ public class EditCommand extends Command {
             + "[" + PREFIX_NAME + "NAME] "
             + "[" + PREFIX_PHONE + "PHONE] "
             + "[" + PREFIX_EMAIL + "EMAIL] "
-            + "[" + PREFIX_ADDRESS + "ADDRESS] "
-            + "[" + PREFIX_TAG + "TAG]... "
+            + "[" + PREFIX_TELEGRAM_HANDLE + "TELEGRAM HANDLE] "
+            + "[" + PREFIX_TAG + "TAG]...\n"
```

Comment:
Is there a specific reason why a new line is added before you introduce the comment parameter?